tests/repro_subagent_anim_leak.py

```
@@ -0,0 +1,163 @@
+                break
+    return data
+
+
+def main() -> int:
+    aish_bin = os.environ.get("AISH_BIN", "target/debug/aish")
+    if not os.path.isfile(aish_bin):
+        print(f"aish binary not found: {aish_bin}")
+        return 2
+
+    real_config_home = os.path.expanduser("~/.config")
+    tmp_config = os.environ.get("XDG_CONFIG_HOME", "/tmp/aish-repro-config")
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/aish-repro-config"

(S108)